include/bluetooth/services/latency.rst

```diff
@@ -0,0 +1,38 @@
+.. _latency_readme:
+
+GATT Latency Service
+####################
+
+The GATT Latency Service is a custom service that receives data and returns result about GATT latency operations.
+
+To measure the GATT transmission latency, when a client writes a timestamp to the latency characteristic on the server, the server will automatically reply back.
+The client can then use its present time and the corresponding timestamp written before to estimate the round-trip time (RTT).
+
+The GATT Latency Service is used in the :ref:`ble_llpm` sample.
```

Comment:
We should probably not have references to examples using the service as this may get outdated

Reply:
I agree. Let's rather have a link from the example to the service.